src/App.tsx

```diff
@@ -15,35 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortBy {
+  Default = '',
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+}
+
+interface FilterParams {
+  sortBy: SortBy;
+  isReversed: boolean;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortBy, isReversed }: FilterParams,
+): string[] {
+  const currentVisibleGoods = [...goods];
+
+  if (sortBy) {
+    currentVisibleGoods.sort((good1: string, good2: string) => {
```

Comment:
Can 'string' be removed? 
or leave it?
 currentVisibleGoods.sort((good1, good2) 

the type is specified on line 31

Reply:
> Can 'string' be removed? or leave it? currentVisibleGoods.sort((good1, good2)
> 
> the type is specified on line 31

Yes, it's unnecessary to set types here, but it's ok